engine/crates/graphql-mocks/src/secure.rs

```diff
@@ -0,0 +1,66 @@
+use async_graphql::{EmptyMutation, EmptySubscription, Object, TypeDirective};
+
+/// A schema that only uses String types.
+///
+/// This is used to make sure that we're not pruning built in scalars that aren't used
```

Comment:
I don't understand this doc comment

Reply:
ah copy pasted and didn't pay attention to the comment, it's irrelevant.